### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by kryptidikettu**
This adds a Dockerfile that can be used to build image that can run SLAMon Python Agent. It is also recommended to create a Docker Hub account (Organization) to where an Automated Build can be created!

Example: https://hub.docker.com/r/stealthyloner/slamon-agent/ (One of the tags reports an error since it tries to build from master)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+RUN groupadd -r slamon && useradd -r -d /workspace -m -g slamon slamon 
+
+# Install dependencies
+RUN pip install requests python-dateutil responses
+
+# Install SLAMon Python Agent
+ADD . /workspace
+WORKDIR /workspace
+RUN pip install .
+RUN chown -R slamon:slamon /workspace
```
Comment (jjonek) [inline Dockerfile]: Should the sources be removed from the image after pip install? Also not sure if a home directory for the user is required, as the agent is currently stateless and not storing anything.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM python:3.4
+
+# Run as non-root, eventually
+RUN groupadd -r slamon && useradd -r -d /workspace -m -g slamon slamon 
+
+# Install dependencies
+RUN pip install requests python-dateutil responses
```
Comment (jjonek) [inline Dockerfile]: I think we can rely on pip to install all the required dependencies when installing the agent later. This way we don't need to keep redundant lists of dependencies.

File: Dockerfile (hunk 1 above)
Comment (kryptidikettu) [inline Dockerfile]: True with the sources, but I think home directory is useful if we want to save logs.

File: Dockerfile (hunk 2 above)
Comment (kryptidikettu) [inline Dockerfile]: True, will fix

**Pull request merged**